apps/fm/js/fm.js

```diff
@@ -619,12 +622,16 @@ var favoritesList = {
     elem.id = this._getUIElemId(item);
     elem.className = 'fav-list-item';
     elem.setAttribute('role', 'option');
-    var html = '';
-    html += '<div class="fav-list-frequency">';
-    html += item.frequency.toFixed(1);
-    html += '</div>';
-    html += '<div class="fav-list-remove-button"></div>';
-    elem.innerHTML = html;
+
+    var subElem = document.createElement('div');
+    subElem.className = 'fav-list-frequency';
+    subElem.textContent = item.frequency.toFixed(1);
+    subElem.dataset.unit = mhzString;
+    elem.appendChild(subElem);
+
+    subElem = document.createElement('div');
+    subElem.className = 'fav-list-remove-button';
+    elem.appendChild(subElem);
```

Comment:
I'm starting to think we should maybe just add a `<span data-l10n-id="mhz"></span>`